docs/gotchas.rst

```diff
@@ -10,8 +10,12 @@ Also see:
   - `PROJ FAQ <https://proj.org/faq.html>`__
 
 
-What is the best format to store the CRS information?
+What are the best formats to store the CRS information?
 -----------------------------------------------------
+
+In general, **Well-Known Text (WKT)** or **Spatial Reference ID (SRID)** are
```

Comment:
I would maybe add something like "(such as EPSG codes)" after the spatial reference ID mention, to make it clearer what this is?

Reply:
Yeah I could add that.  If you're concerned about readers not knowing what SRID means, would links to the relevant Wikipedia pages be appropriate here?

Reply:
Thanks for this 👍. Also, mind adding this note from the PROJ docs: "If possible WKT2 should be used.".

> would links to the relevant Wikipedia pages be appropriate here?

EPSG codes are the most common, so adding the reference to them will likely be sufficient. But, adding a link would be fine if you would like to.

Reply:
Thanks for the feedback, I've updated the PR with those suggestions.

Reply:
Perfect, thanks 👍. Any issues with me squash and merging?

Reply:
Squash is fine by me.